link_test.go

```diff
@@ -5,6 +5,7 @@ import (
 	"net"
 	"testing"
 
+	"github.com/vishvananda/netlink/nl"
```

Comment:
shouldn't need to import nl in the test here once you have made the change below